src/pages/index.tsx

```diff
@@ -1,64 +1,10 @@
 import type { NextPage } from 'next';
-import Head from 'next/head';
-import Image from 'next/image';
 import styles from '../../styles/Home.module.css';
```

Comment:
This css file could also be deleted since the project should from now on only use tailwindcss.